API/Controllers/UploadController.cs

```diff
@@ -225,17 +227,13 @@ public async Task<ActionResult> UploadReadingListCoverImageFromUrl(UploadFileDto
         return BadRequest(await _localizationService.Translate(User.GetUserId(), "generic-cover-reading-list-save"));
     }
 
-    private async Task<string> CreateThumbnail(UploadFileDto uploadFileDto, string filename, int thumbnailSize = 0)
+    private async Task<string> CreateThumbnail(UploadFileDto uploadFileDto, string filename)
     {
-        var encodeFormat = (await _unitOfWork.SettingsRepository.GetSettingsDtoAsync()).EncodeMediaAs;
-        if (thumbnailSize > 0)
-        {
-            return _imageService.CreateThumbnailFromBase64(uploadFileDto.Url,
-                filename, encodeFormat, thumbnailSize);
-        }
-
+        var settings = await _unitOfWork.SettingsRepository.GetSettingsDtoAsync();
+        var encodeFormat = settings.EncodeMediaAs;
+        var coverImageSize = settings.CoverImageSize;
         return _imageService.CreateThumbnailFromBase64(uploadFileDto.Url,
```

Comment:
Please have a newline before a return statement.

Reply:
> Please have a newline before a return statement.

I just found out the size is fixed, but somehow its still low res, it just blows up the low res into a bigger size, i need to have a second look.